src/hotspot/share/prims/jvmti.xml

```diff
@@ -13041,7 +13041,7 @@ myInit() {
   </event>
 
  <event label="Virtual Thread Start"
-         id="VirtualThreadStart" const="JVMTI_EVENT_VIRTUAL_THREAD_START" filtered="thread" num="87" phase="start" since="21">
+         id="VirtualThreadStart" const="JVMTI_EVENT_VIRTUAL_THREAD_START" num="87" phase="start" since="21">
```

Comment:
Does "filtered" mean that the event can be enabled or disabled on a per thread basis, and therefore by removing this it means the event can only be enabled or disabled globally?

Reply:
That's right. The spec for SetEventNotificationMode lists the events cannot be enabled/disabled at the thread level. Both ThreadStart and VirtualThreadStart are listed so I view this JBS/PR issue as fixing the implementation.

Reply:
Thank you for explaining, Alan. There is nothing to add.